### DISCUSSION TITLE
smart screen issue with signing setup

### DISCUSSION THREAD

**Discussion by ghost1372** [microsoft/winget-pkgs#1955] [General] [answered]
Hi, I am signing the setup with the pfx file, but The user must install the signature on their system before installing the setup so that they do not face the problem of smart screen.
Is there a suitable solution? I have this problem with MSIX, MSI and exe setup.

**Comment by KevinLaMS** [comment, marked as answer]
@ghost1372 Yeah, signing can be tough.  We 'may' have a short term solution and a longer term solution.  
For the longer term solution we are looking at standing up a signing service.  See  https://twitter.com/jvintzel/status/1262794128351322114

Short term, we are currently working with smartscreen to help establish reputation on submitted apps.  If you are not frequently updating, then this process may work.
Let's see if this process works for you.  Submit your manifest.  Then send me email directly with your PR and I will track it through the validation steps.  Kevinla@Microsoft.com

**Comment by ghost1372** [comment]
Thank you
